app/helpers/shf_applications_helper.rb

```diff
@@ -75,4 +75,34 @@ def reasons_for_waiting_info(method_prefix, locale)
     AdminOnly::MemberAppWaitingReason.all.map { |r| [r.id, r.send(method_name)] }
   end
 
+  def file_delivery_radio_buttons_collection(locale = I18n.locale)
```

Comment:
By "needs to be a partial" I assume you mean the view where this is used.  I'd rather not do that since that view section is only used in one place, and I'd rather see the section layout in context of the containing form.

It looks like this, which I don't think merits a partial:

```
%span.file-delivery-selection
  Please select how you will deliver the files needed for your application:
  %br

  - radio_buttons_collection, footnotes = file_delivery_radio_buttons_collection

  = f.collection_radio_buttons(:file_delivery_method_id,
                                radio_buttons_collection,
                                :first, :second, {}) do |rb|
    = rb.label(style: 'margin-left: 10px;', class: 'radio-inline') do
      = rb.radio_button + ' ' + rb.text
%br
%span.file-delivery-footnotes
  = footnotes
```

Reply:
makes sense. (so no partial)